vms/avm/vm_test.go

```diff
@@ -218,10 +226,19 @@ func BuildGenesisTest(t *testing.T) []byte {
 	}
 
 	return hex.Bytes
+
 }
 
 func GenesisVM(t *testing.T) ([]byte, chan common.Message, *VM, *atomic.Memory) {
+	return GenesisVMWithArgs(t, nil)
+}
+
+func GenesisVMWithArgs(t *testing.T, args *BuildGenesisArgs) ([]byte, chan common.Message, *VM, *atomic.Memory) {
 	genesisBytes := BuildGenesisTest(t)
+	if args != nil {
```

Comment:
Should `BuildGenesisTest` only be called if `args == nil`?

Reply:
Yeah - doesn't make a load of sense of duplicating the call. I thought it only setup args, but I see it calls the `CreateStaticService` and the `BuildGenesis` after 👍